config_files/survey_year_2019_config.yml

```diff
@@ -4,94 +4,81 @@
-  haul_to_transect_CAN_sheetname: Sheet1
-  ##############################################################################
+##############################################################################
+# Parameters
+
+survey_year: 2019            # survey year being considered
+species_id: 22500            # target species ID for the survey year
+CAN_haul_offset: 200         # The value to be added to the Canadian's haul number
+hemisphere: NW               # north/south and west/east hemispheres, used for the reports
+
+##############################################################################
+# Directory path that contains all input data needed
+
+data_root_dir: /usr/mayorgadat/workmain/acoustics/2021-NWFSC-EchoPro-HakeIGP/EchoPro/EchoProPython/2019_consolidated_files
+#  data_root_dir: ../../2019_consolidated_files/
+
+##############################################################################
+# Input data files
+
+biological:
+  length:
+    US:
+      filename: Biological/US/2019_biodata_length.xlsx
+      sheetname: biodata_length
+    CAN:
+      filename: Biological/CAN/2019_biodata_length_CAN.xlsx
+      sheetname: biodata_length_CAN
+  specimen:
+    US:
+      filename: Biological/US/2019_biodata_specimen_AGES.xlsx
+      sheetname: biodata_specimen
+    CAN:
+      filename: Biological/CAN/2019_biodata_specimen_CAN_AGES.xlsx
+      sheetname: biodata_specimen_CAN
+  catch:
+    US:
+      filename: Biological/US/2019_biodata_catch.xlsx
+      sheetname: biodata_catch
+    CAN:
+      filename: Biological/CAN/2019_biodata_catch_CAN.xlsx
+      sheetname: biodata_catch_CAN
+  haul_to_transect:
+    US:
+      filename: Biological/US/haul_to_transect_mapping_2019.xlsx
+      sheetname: Sheet1
+    CAN:
+      filename: Biological/CAN/haul_to_transect_mapping_2019_CAN.xlsx
+      sheetname: Sheet1
+stratification:
+  strata:
+    # The two stratification types are found in two sheets: "Base KS" and "INPFC"
+    filename: Stratification/US_CAN strata 2019_final.xlsx
+    sheetname: Base KS
+  geo_strata:
+    # The two stratification types are found in two sheets: "stratification1" and "INPFC"
+    filename: Stratification/Stratification_geographic_Lat_2019_final.xlsx
+    sheetname: stratification1
+NASC:
+  # NASC values
+  no_age1:
+    # file that excludes age1 values
+    filename: Exports/US_CAN_detailsa_2019_table2y+_ALL_final - updated.xlsx
+    sheetname: Sheet1
+  all_ages:
+    # file that includes all ages
+    filename: Exports/US_CAN_detailsa_2019_table1y+_ALL_final - updated.xlsx
+    sheetname: Sheet1
+kriging:
+  mesh:
+    filename: Kriging_files/Kriging_grid_files/krig_grid2_5nm_cut_centroids_2013.xlsx
+    sheetname: krigedgrid2_5nm_forChu
+  smoothed_contour:
+    # filename: Kriging_files/Kriging_grid_files/Smoothing_EasyKrig.xlsx
+    filename: Kriging_files/Kriging_grid_files/transformation_isobath_coordinates.xlsx
+    sheetname: Smoothing_EasyKrig
```

Comment:
Does this file contain the coordinate of the 250m isobath? Thinking if that's the case, maybe we can change the yaml item name from `smoothed_contour` to `isobath_250m` or something similar.

Reply:
> Does this file contain the coordinate of the 250m isobath? 

Correct (see [here](https://uw-echospace.github.io/EchoPro/input-files.html#smoothed-shelf-break-contour)), except my understanding from the Hake Survey Methods document is that it's the 200m isobath, not 250m. 

Great suggestion! The name "smoothed contour" is unhelpful, but it's used widely, unfortunately. I agree with changing it here, in this user-facing context, and I like your suggested name -- So, `isobath_200m`?

Down the line, I think it'd be good to change variable names in the code to move away from "smooth contour". But let's do that in a separate PR.

Reply:
I've pushed a commit implementing the change to `isobath_200m`

Reply:
Sounds good!